frontend/schema.faker.graphql

```diff
@@ -1090,3 +1101,66 @@ type WWWScanEdge {
   """A cursor for use in pagination"""
   cursor: String!
 }
+
+"""
+This custom object is used to populate the userList componenet in the front end.
+"""
+type UserList {
+  """ Indicates which organization this list is being queried for."""
+  organization: Acronym!
+
+  """Pagination data for this connection."""
+  pageInfo: PageInfo!
+
+  """Contains the nodes in this connection.  Aka: A list of userItems that will make up the list on the front end."""
+  edges: [UserItemEdge]!
+}
+
+type UserListItem implements Node {
+    """The ID of the object."""
+    id: ID!
+
+    """ The users email address or userName """
+    userName: EmailAddress! @fake(type: email)
+
+    """ The users display name"""
+    displayName: String! @fake(type: firstName)
+
+    """ Indicates wether or not this user has enabled two factor authentication"""
+    tfa: Boolean! @examples(values: [true, false])
+
+    """ Indicates if this user is an admin of the organization specified in UserList query."""
+    admin: Boolean! @examples(values: [true, false])
+}
+
+"""
+This custom gql object is used to populate a userPage component in the front end.
+"""
+type UserPage{
+    """ The users email address or userName"""
+    userName: EmailAddress! @fake(type: email)
+
+    """ The users display name"""
+    displayName: String! @fake(type: firstName)
+
+    """ Indicates which organization this users data is being displayed for."""
+    organization: Acronym! @examples(values: ["GC", "ABC", "ASDF", "NSTIR", "BC"])
```

Comment:
Just wondering if org and admin can be moved into its own type, and added as a list, just in case they have multiple affiliations and the admin seeing also has those affiliations, also they may be admin in multiple orgs